relayer/provider/provider.go

```diff
@@ -68,7 +69,7 @@ type QueryProvider interface {
 	QueryUpgradedClient(height int64) (*clienttypes.QueryClientStateResponse, error)
 	QueryUpgradedConsState(height int64) (*clienttypes.QueryConsensusStateResponse, error)
 	QueryConsensusState(height int64) (ibcexported.ConsensusState, int64, error)
-	QueryClients() ([]*clienttypes.IdentifiedClientState, error)
+	QueryClients() ([]clienttypes.IdentifiedClientState, error)
```

Comment:
y no pointer?

Reply:
The returned data, `IdentifiedClientState` is exposed via a struct found in `ibc-go` which is an array of structs vs. array of pointers

https://github.com/cosmos/relayer/blob/5d02ee2710ab673dbc8e4661f03572d82d9b8389/relayer/provider/cosmos/provider.go#L734

```
// QueryClientStatesResponse is the response type for the Query/ClientStates RPC
// method.
type QueryClientStatesResponse struct {
	// list of stored ClientStates of the chain.
	ClientStates IdentifiedClientStates `protobuf:"bytes,1,rep,name=client_states,json=clientStates,proto3,castrepeated=IdentifiedClientStates" json:"client_states"`
	// pagination response
	Pagination *query.PageResponse `protobuf:"bytes,2,opt,name=pagination,proto3" json:"pagination,omitempty"`
}
```